components/HorizontalScroller.js

```diff
@@ -1,68 +1,57 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { ChevronLeft } from '@styled-icons/fa-solid/ChevronLeft';
-import { ChevronRight } from '@styled-icons/fa-solid/ChevronRight';
-import themeGet from '@styled-system/theme-get';
+import { ArrowBack } from '@styled-icons/material/ArrowBack';
+import { ArrowForward } from '@styled-icons/material/ArrowForward';
 import styled from 'styled-components';
 
 import { debounceScroll } from '../lib/ui-utils';
 import withViewport from '../lib/withViewport';
 
-/** Main chevrons container, spacing the items properly */
-const ChevronsContainer = styled(({ onPrevClick, onNextClick, ...props }) => (
-  <div {...props}>
-    <ChevronLeft onMouseDown={onPrevClick} disabled={!onPrevClick} />
-    <ChevronRight onMouseDown={onNextClick} disabled={!onNextClick} />
-  </div>
-))`
+import { Box, Flex } from './Grid';
+
+const RefContainer = styled(Box)`
   display: flex;
-  justify-content: space-between;
-  color: #dadada;
-  user-select: none;
-  width: ${props => props.size * 3.5}px;
+  overflow-x: auto;
+  scroll-behavior: smooth;
+`;
 
-  svg {
-    width: ${props => props.size}px;
-    height: ${props => props.size}px;
+const ControlsContainer = styled(Flex)`
+  z-index: 10;
+  position: relative;
+  top: 30rem;
+`;
 
-    &:not([disabled]) {
-      color: ${themeGet('colors.primary.500')};
+const ArrowContainer = styled.button`
+  appearance: none;
+  border: none;
+  background-color: transparent;
 
-      &:hover {
-        cursor: pointer;
-        color: ${themeGet('colors.primary.400')};
-      }
+  svg {
+    height: 40px;
+    padding 7px;
+    vertical-align: middle;
+    border: 1px solid #C4C7CC;
+    border-radius: 50%;
+    background-color: #FFFFFF;
+    transition: opacity 0.25s ease-in;
+    opacity: ${props => (props.disabled ? '0' : '1')};
```

Comment:
This is not enough, as the button stays clickable. We need to make sure it cannot be (by removing the node, or with a `display: none` for example).

![Peek 2021-04-21 15-17](https://user-images.githubusercontent.com/1556356/115560661-18930280-a2b5-11eb-9c26-d9219777388c.gif)

Reply:
How about using `visibility: hidden`? it won't consume clicks.

![arrow](https://user-images.githubusercontent.com/46647141/115564207-e645cd00-a2d5-11eb-9339-7b6d8f9d0d3d.gif)

Reply:
How about manipulating `visibility: hidden` too? I have also used `StyledRoundButton` in the gif below.

![arr](https://user-images.githubusercontent.com/46647141/115566584-fd85ba00-a2d7-11eb-9378-510ff588beb8.gif)

Reply:
That can work, as long as we're still able to click a link below the button it's all good.

In the preview you shared, the button colors still don't seem to match the design system. We can normally remove custom styles from there, and just use `StyledRoundButton`'s defaults.

Reply:
Updated 👍🏻